src/Diffract/CustomDiffer.fs

```diff
@@ -0,0 +1,107 @@
+﻿namespace DEdge.Diffract
+
+open System
+open DEdge.Diffract
+
+[<Sealed; AbstractClass>]
+type CustomDiffer<'T> =
```

Comment:
There's definitely a bunch of dodgy naming throughout the library. For now this static class is named `CustomDiffer` because it deals with values of type `ICustomDiffer`; but at some point (at or before v1.0) we should do a full pass of renaming.